protocol/0044-LIME-lp_mechanics.md

```diff
@@ -330,10 +330,10 @@ In the case of spot markets it will be transferred into the network treasury for
   `LP2` will receive a full `100` stake back whilst `LP1` will receive a total of `85` back into their general account with `15` transferred into the market's insurance account  (<a name="0044-LIME-026" href="#0044-LIME-026">0044-LIME-026</a>)
 - When LP is committed they are obliged to provide liquidity equal to their commitment size on both sides of the order book (<a name="0044-LIME-027" href="#0044-LIME-027">0044-LIME-027</a>)
 - For a market that is in opening auction and LP has committed provision:
-  - LP can increase their provision and it will take affect immediately (<a name="0044-LIME-050" href="#0044-LIME-050">0044-LIME-050</a>)
-  - LP can decrease their provision and it will take affect immediately without incurring penalties (<a name="0044-LIME-051" href="#0044-LIME-051">0044-LIME-051</a>)
-  - LP cannot cancel their commitment but instead submit governance vote to cancel the market (<a name="0044-LIME-052" href="#0044-LIME-052">0044-LIME-052</a>)
-  - If multiple LPs committed provision then an LP can cancel their commitment by decreasing to `0` commitment without incurring penalties (<a name="0044-LIME-053" href="#0044-LIME-053">0044-LIME-053</a>)
+  - LP can increase their provision and it will take affect immediately (<a name="0044-LIME-050" href="#0044-LIME-050">0044-LIME-050</a>) for spots (<a name="0044-LIME-054" href="#0044-LIME-054">0044-LIME-054</a>)
+  - LP can decrease their provision and it will take affect immediately without incurring penalties (<a name="0044-LIME-051" href="#0044-LIME-051">0044-LIME-051</a>) for spots (<a name="0044-LIME-055" href="#0044-LIME-055">0044-LIME-055</a>)
```

Comment:
```suggestion
  - LP can decrease their provision and it will take affect immediately without incurring penalties (<a name="0044-LIME-051" href="#0044-LIME-051">0044-LIME-051</a>) for spot (<a name="0044-LIME-055" href="#0044-LIME-055">0044-LIME-055</a>)
```